src/domain/interfaces/auth-api.interface.ts

```diff
@@ -9,4 +9,10 @@ export interface IAuthApi {
     message: SiweMessage;
     signature: `0x${string}`;
   }): Promise<boolean>;
+
+  cacheNonce(nonce: string): Promise<void>;
```

Comment:
We should avoid mentioning the cache backend (if possible). We can then change the backend without changing the interface. 

```suggestion
  storeNonce(nonce: string): Promise<void>;
```

Reply:
Changed in b15fa98e39c07d2987f9f21eafce4ea6e318ec21.